tests/e2e/client/apiproxy_bookstore_test.py

```diff
@@ -152,6 +152,18 @@ def verify_list_books(self, shelf, books):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json().get('books', []), books)
 
+    def verify_swagger_importer(self):
```

Comment:
I don't understand the naming here. What is "swagger importer"?

Reply:
swagger_importer is the actual name of api compiler.